shared/common-adapters/markdown.shared.js

```diff
@@ -48,6 +48,7 @@ function preprocessMarkdown(markdown: string, meta: ?MarkdownMeta) {
     return markdown
   }
 
+  // TODO: Allow uppercase in mentions, and just normalize.
   return markdown.replace(/\B@([a-z0-9][a-z0-9_]+)/g, (match, matchedGroup) => {
```

Comment:
This regexp is inconsistent with the others, but this will go away in another PR.